roles/h-asgi/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install h-asgi systemd file
+  include_role:
+    name: podman
+  vars:
+    - container:
+        name: h-asgi
+        image: ghcr.io/vcokltfre/h:latest
+        mem_max: 25M
+        cpu_quota: 3%
+        port: 8000:8000
```

Comment:
Wait, how does this work with a list..??

Reply:
Honestly, i have no idea. But it does :D